source/publishing_data/05_processing_data/processors/retrieve_administrative_divisions.rst

```diff
@@ -58,7 +55,7 @@ To set the parameters of the Retrieve Administrative Divisions processor, follow
 Referentials
 ------------
 
-When a real country is chosen, the administrative levels corresponding to that country are explicitely written. However, when using the 'World' country, an administrative level between 1 and 7 must be chosen. Please refer to the table below to have an idea of what the administrative levels are for each country currently available in the processor.
+When a real country is chosen, the administrative levels corresponding to that country are explicitly written. However, when using the 'World' country, an administrative level between 1 and 7 must be chosen. Please refer to the table below to have an idea of what the administrative levels are for each country currently available in the processor.
```

Comment:
It’s not clearly in the scope of the review, but I’m puzzled by the "administrative level between 1 and 7" to provide as a parameter. Since we revamped the geo referentials last year (and maybe before), the levels are between 20 (world) and 99 for the smallest units (we only go to 92 for now).

When using the processor, I find a mix of texts (names of same-level divisions for various countries) and numerical levels that have not been described…
<img width="349" alt="Capture d’écran 2021-07-29 à 17 21 42" src="https://user-images.githubusercontent.com/52446951/127519372-4c1c9402-c0b8-4f39-a69b-a01e20401bbf.png">

Could the doc be clarified to explain that the administrative level to be chosen is either a textual depiction of the content of the level (names of divisions) or a number, and that the possibilities of the list are ordered from the bigger (above) to the smallest level (below)?

Reply:
I created a story so that we can improve the docs for this processor: https://app.clubhouse.io/opendatasoft/story/29390/as-a-user-i-need-to-have-better-docs-for-the-retrieve-administration-divisions-processor